### ISSUE TITLE
No icons

### ISSUE THREAD

**Issue by BarbzYHOOL** [AmjadHD/sublime_one_theme#7]
I don't have cool icons like in ur screenshot

I have the package "fileicons" installed and am only satisfied with the adaptive theme so far, there is always something wrong

**Comment by AmjadHD** [comment]
tl;dr just install [A File Icons](https://packagecontrol.io/packages/A%20File%20Icon).

There are problems with File Icons:
* The FileIcons package by default only supports the `Default` and `Adaptive` themes.
* Enabling it without customization hides the shipped icons and doesn't show its own.
* If you follow the steps in https://github.com/braver/FileIcons#enable-file-icons-for-3rd-party-themes to enable FileIcons for this theme, the icons will be activated but dimmed:
![image](https://user-images.githubusercontent.com/33235928/201516356-438e7da7-6138-46db-a5e7-ccffec633987.png)
because ST merges themes by lexical order, the rules in `FileIcons/theme/One.sublime-theme` will be overridden by the rules in the original `Theme - One/One.sublime-theme` which dim the icons shipped with the theme.
You can remedy this by customizing the theme (for ST4, Run `UI: Customize Theme` from the command palette, for ST3, navigate to `Preferences > Package Settings > Theme - One > Customize`.), and add these rules:
```jsonc
{
  {
    "class": "icon_file_type",
    "layer0.tint": null,
    "layer0.opacity": 0.9,
    "content_margin": [8, 8]
  },
  {
    "class": "icon_file_type",
    "layer0.tint": null,
    "parents": [{"class": "tree_row", "attributes": ["selected"]}],
    "layer0.opacity": 1.0
  }
}
```
* But since `FileIcons` doesn't ship with all icons, the shipped icons end up without tint:
![image](https://user-images.githubusercontent.com/33235928/201516223-39e308ae-3102-42df-bbae-c2851202b6e4.png)
So just disable `FileIcons` and install `A File Icons` instead.

**Comment by BarbzYHOOL** [comment]
thanks, it worked and i learned a few things doing it
